### DISCUSSION TITLE
Non ASCII symbols in ggplot2 plots in beamer

### DISCUSSION THREAD

**Discussion by agricolamz** [quarto-dev/quarto-cli#8696] [Q&A]
### Bug description

When I use non ASCII symbols in `ggplot2` plots in my beamer presentation I got dots. The same plot is displayed correctly on my system. Changing fonts with the package `extrafonts` doesn't help: the same dots.

### Steps to reproduce

``````
---
format: beamer
---

## test

```{r}
#| echo: false

library(tidyverse)
iris |> 
  ggplot(aes(Sepal.Length, Sepal.Width))+
  geom_point()+
  labs(x = "Добър", y = "ден")
```
``````
### Expected behavior

I expected cyrillic letters to appear on x and y axes.

### Actual behavior

All cyrillic symbols are changed into dots.

![image](https://github.com/quarto-dev/quarto-cli/assets/15956515/5c427318-6b54-488d-88ed-5957bc7fabcd)

### Your environment

- IDE: RStudio 2023.12.1
- OS: Linux Mint 21.3 x86_64

### Quarto check output

Quarto 1.4.549

**Comment by cscheid** [comment]
This is a support request for knitr/ggplot2 help, so I converted it to one. Please keep it in mind for future report. Thanks!
